### ISSUE TITLE
Setting the root folder for the music directory to a group(sub?)folder leads to no files in collection

### ISSUE THREAD

**Issue by anthraxn8b** [owncloud/music#844]
If I open the music settings of a user and point to a folder that is located in a group folder no music files are found.

**Comment by paulijar** [comment]
What is a group folder? Some folder shared to a group? Does it maybe reside on some external storage? I was wondering if you are suffering from an issue similar to https://github.com/owncloud/music/issues/812.

**Comment by ctc** [comment]
Music in shared folders is not found.

**Comment by anthraxn8b** [comment]
However this is called in english (I have the german translation set...) - maybe "shared folders" - but it is a plugin.
However. It is working (music is listed) if no explicit path for media is set.
I suggest to have a look at the group/shared folder name resolution...

**Comment by paulijar** [comment]
I installed Group Folders 5.0.8 on NC17 and Group Folders 9.0.0 on NC21. On both environments, I created one group folder and uploaded 3 mp files to it. In both environments, after changing the music library path to the group folder, all 3 files were found and scanned just fine.

However, on the NC17, the scanning process got stuck and the scanned files didn't appear before I reloaded the page. Looking at the nextcloud.log file, it was revealed that the process had exhausted the PHP memory quota of 256MB. This probably happened when looking for the artist cover images as the files themself were scanned properly. Now, I don't know if this problem happened only in this environment because of the older NC version, the older Group Folders version, or the fact that this cloud instance contained a lot more content on many users than the NC21 environment which had only a few small test accounts.

Which versions are you using? Is there anything in nextcloud.log?

**Comment by anthraxn8b** [comment]
Strange. I use NC 20. Maybe a german “Umlaut“ is part of my problem - I already removed it without success... I will investigate further... THX for testing!

**Comment by hobbsecky** [comment]
I have NC 21.0.1 with the Group Folders 9.0.1 addon. Everything works as expected for me.

Logged into Nextcloud with my Admin account I click Users>Add Group> Create a group named "Users"

Next I go to the Admin Settings.

Using the Nextcloud Registration App I select the "Registered Users Default Group" and add the "Users" group I created earlier.

Using the Nextcloud Group Folders App I create a folder "Music Share". I then add "Users" and "Admin" to the "Groups" and I add "Admin" to "Advanced Permissions". I set the "User" Permission to Write and Share only and Admin to Write, Share, Delete. This folder links to a massive Music share that I have on a separate drive.

When the new user opens the Music app  "New music found" appears and they click it and everything works as expected.
